### PR TITLE
[zwavejs2mqtt] Update image and add websocket port example

### PR DESCRIPTION
**Pull request by Ardetus**
**Description of the change**

**Benefits**

**Possible drawbacks**

**Applicable issues**

- fixes #

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [ ] (optional) Chart is using our [common](https://github.com/k8s-at-home/charts/tree/master/charts/common) library as a dependency.
- [ ] (optional) Variables are documented in the README.md

### CODE REVIEW

File: charts/stable/zwavejs2mqtt/values.yaml [hunk 1]
```diff
@@ -21,6 +21,9 @@ env: {}
 service:
   port:
     port: 8091
+  # additionalPorts:
+  # ┊ - name: websocket
+  # ┊   port: 3000
```
Comment (onedr0p) [inline charts/stable/zwavejs2mqtt/values.yaml:26]: Are you sure you don't want to add an `additionalService` instead, would a `additionalPort` be fine for this usecase?

**Review by onedr0p — COMMENTED** [review]
Which one should we use here.

File: charts/stable/zwavejs2mqtt/values.yaml (hunk 1 above)
Comment (Ardetus) [reply charts/stable/zwavejs2mqtt/values.yaml]: Not really sure what is proper way... I copied this from deconz chart and it worked fine on my setup

Comment (onedr0p) [reply charts/stable/zwavejs2mqtt/values.yaml]: Sounds good, we can roll with it :)

**Review by onedr0p — APPROVED** [review]

**Comment by onedr0p** [comment]
@all-contributors add @Ardetus for code

**Pull request merged**